dir.targets

```diff
@@ -36,6 +36,8 @@
     <XunitOptions Condition="'$(OS)'=='Unix'">$(XunitOptions) -notrait category=nonlinuxtests</XunitOptions>
     <XunitOptions Condition="'$(OS)'=='OSX'">$(XunitOptions) -notrait category=nonosxtests</XunitOptions>
     <XunitOptions Condition="'$(OS)'=='OSX' and '$(NetCoreBuild)'=='true'">-notrait category=nonosxtests -notrait category=netcore-osx-failing</XunitOptions>
+    <XunitOptions Condition="'$(OS)'=='Unix' and '$(NetCoreBuild)'=='true'">-notrait category=nonlinuxtests -notrait category=netcore-linux-failing</XunitOptions>
```

Comment:
Use `$(XunitOptions)` - 
`<XunitOptions Condition="'$(OS)'=='Unix' and '$(NetCoreBuild)'=='true'">$(XunitOptions) -notrait category=netcore-linux-failing</XunitOptions>` 

And same for the `OSX && netcore` case.